packages/desktop-client/src/components/budget/BudgetSummaries.js

```diff
@@ -89,15 +69,17 @@ export default function BudgetSummaries({ SummaryComponent }) {
       ])}
       ref={containerRef}
     >
-      <View
+      <animated.div
+        className="view"
```

Comment:
Do we need this `className`?

Reply:
Yes, `View` applies some CSS styles and I didn’t want to lose them